doc/source/train/common/torch-configure-run.rst

```diff
@@ -84,3 +84,14 @@ information about the training run, including the metrics and checkpoints report
     result.error       # The exception that was raised, if training failed.
 
 For more usage examples, see :ref:`train-inspect-results`.
+
+Ray Debugger
```

Comment:
Do we intend to use `Ray Debugger` as the title here?

Reply:
there was an issue with "Ray Distributed Debugger" because the title was too long but so I just fell back to "Ray Debugger"